src/options/quantifiers_options.toml

```diff
@@ -505,44 +505,52 @@ name   = "Quantifiers"
   long       = "full-saturate-quant"
   type       = "bool"
   default    = "false"
+  help       = "resort to full effort techniques instead of answering unknown due to quantifiers. Currently enables enumerative instantiation"
```

Comment:
```suggestion
  help       = "resort to full effort techniques instead of answering unknown due to limited quantifier reasoning. Currently enables enumerative instantiation"
```